Objects/Converters/ConverterRevit/ConverterRevitShared/PartialClasses/ConvertGeometry.cs

```diff
@@ -1259,7 +1259,12 @@ public Brep BrepToSpeckle(Solid solid, Document d, string units = null)
             // Update trim indices with new item.
             // TODO: Make this better.
             var trimIndices = sEdge.TrimIndices.ToList();
-            trimIndices.Append(sTrimIndex); //TODO Append is a pure function and the return is unused
+
```

Comment:
I think I saw this before? Not sure why we supress rather than fix?

Reply:
It's within a piece of code that is never used in any execution path, as we don't support `BrepToSpeckle` from Revit.

We could fix it, but I can't remember why on earth I did this and what was I thinking at the time, so I feel better if we just leave it as is, and pick it up if we ever tackle Revit Solid to Brep conversion (there's also a ticket for that deep in the backlog)